PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/storage/component/VMStorageAccountCreationDialog.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.storage.component;
+
+import com.intellij.openapi.project.Project;
+import com.microsoft.azure.toolkit.lib.storage.model.Kind;
+import com.microsoft.azure.toolkit.lib.storage.model.Performance;
+import com.microsoft.azure.toolkit.lib.storage.model.Redundancy;
+
+import java.util.Arrays;
+
+public class VMStorageAccountCreationDialog extends BaseStorageAccountCreationDialog {
+
+    public VMStorageAccountCreationDialog(Project project) {
+        super(project);
+        this.initialize();
+    }
+
+    private void initialize() {
```

Comment:
initialize -> init ?

Reply:
dialog has a `init` method. this naming is order to avoid conflict.